02LessonTwo/homework/index.html

```diff
@@ -1,10 +1,82 @@
 <!DOCTYPE html>
 <html>
 <head>
-  <title></title>
+  <title>Akshaya's Mini Bio</title>
   <link rel="stylesheet" href="css/style.css">
 </head>
 <body>
+<h1 align="Center">Akshaya Saraf</h1>
+<h5 align="Center" ><i>13224 Appaloosa Chase Drive,Austin, TX - 78732 </i><hr> Now you know where to find me! </hr></h5>
+<div> <img src="https://media.licdn.com/mpr/mpr/shrinknp_200_200/AAEAAQAAAAAAAAawAAAAJDU5YzhlODA2LTg5YjktNGE1Yy05MDMyLTc4Yjc3MTE1ZDRjZA.jpg" alt="Akshaya's mini Bio" class="Center"> </div>
+<h5 align="Center">Bio</h5>
+<p align= "Center">
+<pre>
```

Comment:
we dont really use this 'pre' tag. we use a separate CSS file to style everything.